components/browser/awesomebar/src/main/res/layout/mozac_browser_awesomebar_item_generic.xml

```diff
@@ -10,6 +10,7 @@
     android:background="?android:attr/selectableItemBackground"
     android:clickable="true"
     android:focusable="true"
+    android:clipToPadding="false"
```

Comment:
I'm confused. Doesn't this mean we draw the icon over the padding we defined? Is that really the right fix here?

Reply:
I could swap padding with margin. Is that better?

Reply:
There are other options that I have taken into consideration that have some tradeoffs:
1. Swap padding with margin - no visual difference, except talkback rectangle will be over the icon in both RTL and LTR
2. Change either paddingStart or paddingEnd to have the same value - the padding will change in both RTL and LTR

I thought that having a smaller padding (because we draw the icon over the padding) just in RTL was the one with the smallest visual change. If you think that another solution is more acceptable, please let me know, I will be happy to try it. TY!